packages/ui/src/proposals/components/VotesPreview/VotesPreview.tsx

```diff
@@ -12,15 +13,16 @@ const { Approve, Reject, Slash, Abstain } = ProposalVoteKind
 
 interface VotesPreviewProps {
   votes: VotingRound
+  notVoted?: Member[]
 }
 
-export const VotesPreview = ({ votes }: VotesPreviewProps) => (
+export const VotesPreview = ({ votes, notVoted }: VotesPreviewProps) => (
   <VotesContainer>
     <VotePreview kind={Approve} count={votes.count.approve} votes={votes.map.get(Approve)} />
     <VotePreview kind={Reject} count={votes.count.reject} votes={votes.map.get(Reject)} />
     <VotePreview kind={Slash} count={votes.count.slash} votes={votes.map.get(Slash)} />
     <VotePreview kind={Abstain} count={votes.count.abstain} votes={votes.map.get(Abstain)} />
-    <VotePreview kind="Not Voted" count={votes.count.remain} />
+    <VotePreview kind="Not Voted" count={notVoted?.length ?? 0} votes={notVoted?.map((voter) => ({ voter })) ?? []} />
```

Comment:
One last thing:
```suggestion
    <VotePreview kind="Not Voted" count={notVoted?.length ?? votes.count.remain} votes={notVoted?.map((voter) => ({ voter })) ?? []} />
```
I think defaulting to `votes.count.remain` is a better temporary solution. Because showing `0` is (most of the time) a false information [e.g here](https://dao-git-fix-not-voted-section-joystream.vercel.app/#/proposals/preview/3). While `votes.count.remain` should be correct at least until we address #2941 (i.e the size of the council shouldn't change before that)